tests/checkers/unittest_unicode/bad_chars.py

```diff
@@ -0,0 +1,255 @@
+import itertools
+from pathlib import Path
+from typing import Tuple, cast
+
+import astroid
+import pytest
+from astroid import AstroidBuildingError, nodes
+
+import pylint.checkers.unicode
+import pylint.interfaces
+import pylint.testutils
+
+from . import CODEC_AND_MSG, FakeNode
```

Comment:
We should use pytest.fixtures in a ``conftest.py`` for that

Reply:
Using a fixture for CODEC_AND_MSG is complicated as I want to combine them in in `pytest.mark.parameterize`.
I don't say it is impossible but would increase complexity and I would need to invest time into learning it for something that makes to code more complicated....

Reply:
No we need the relation between both. At least I can't think of a simple and reliable way.

There are possibilites in pytest to achieve all this properly but yet again, they are not easy and I don't want spend that time just so that there are no CONSTANTS in an test script....

Reply:
Fair enough